js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,12 @@
+function duplicateLetters(...args) {
+  const result = Object.values([...args[0]].reduce((obj, val) => {
```

Comment:
May be consider argument to have multiple entries and not just 1

Reply:
Okay, Thanks. I will update